readthedocs/rtd_tests/tests/test_doc_building.py

```diff
@@ -1192,7 +1191,6 @@ def setUp(self):
         self.build_env_mock = Mock()
 
         self.base_requirements = [
-            'setuptools',
```

Comment:
Are we checking in another place of the test that we are installing `setuptools==41.0.1`? I think there should be an update for a test case, right?